runtime/doc/treesitter.txt

```diff
@@ -352,6 +370,18 @@ attribute: >
 ==============================================================================
 Lua module: vim.treesitter                               *lua-treesitter-core*
 
+                                                  *get_captures_at_position()*
```

Comment:
If it does return highlight groups as the docstring indicates, `get_hlgroups_at_position()` would be a better name?